js/cloudmine.js

```diff
@@ -1479,9 +1490,19 @@
       'X-CloudMine-UT': opts.user_token
     };
 
+      /**
+       * I am adding an else here to account for the adding back the old set webservice method. I don't know if the global
+       * object is even used anymore but if for some strange reason the xuniqueid is not set in the option
+       * we can possibly get it from the global obj ... if global is set
+       */
     if (opts.xuniqueid) {
       this.requestHeaders['X-Unique-Id'] = opts.xuniqueid;
     }
+    else {
```

Comment:
Can you just move this else up to the line above next to the closing bracket (`} else {`). That is idiomatic syntax.